### ISSUE TITLE
Sorting by name?

### ISSUE THREAD

**Issue by gpriestley** [stevendewaele/MarketingListManager#3]
Any chance this could be updated so that the grid of Marketing Lists is sorted by the Name column?
With 100's of marketing lists, this becomes completely unusable in the random order that it's currently displayed.

It would also be good if it retrieved just Active Marketing Lists (statecode=0) to filter out the disabled ones.

**Comment by ghost** [comment]
Firstly, thanks for making this - it's a very useful tool. I haven't found another tool on XRM Toolbox that does this.

I completely agree on this request. The option to filter or sort data via the main table interface would be exceptionally useful. I have just had to manually scan down a list of hundreds of marketing lists to find the two I needed to fetch counts for.

**Comment by gpriestley** [comment]
I had a developer friend of mine add this capability.  We've been meaning to catch up to finish it off and work out how to push the code back into the public domain and somehow merge it back with the original distribution, but I can provide the DLL that you need if you're interested.

**Comment by ghost** [comment]
> I had a developer friend of mine add this capability. We've been meaning to catch up to finish it off and work out how to push the code back into the public domain and somehow merge it back with the original distribution, but I can provide the DLL that you need if you're interested.

Yes that would be amazing thank you! I am guessing you could fork this and then add in the new DLL? It looks like you can add files to comments to that likely is the best option so others can see and grab it.

**Comment by gpriestley** [comment]
Usual disclaimers - all care no responsibility.
ZIP of DLL here:
https://ln5.sync.com/dl/e389eaae0/khjexqcy-dp94vukj-v3x4uwa9-7zjxpv7x

Just need to replace existing instances of the DLL under the XRMToolbox install directory.

**Comment by ghost** [comment]
Thank you for that - works a treat!
